sqrt_function_algorithm/sqrt_cplusplus.cpp

```diff
@@ -0,0 +1,32 @@
+#include<iostream.h>
+#include<conio.h>
```

Comment:
Also, I think you do programming in Turbo C, but conio.h is not a standard library for all of the platforms, so can you remove this line. Also, remove ```clrscr();```, and ```getch();```